Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc

```diff
@@ -270,10 +276,11 @@ void DDHGCalEEAlgo::positionSensitive(DDLogicalPart& glog, double rin,
   double dy   = 0.75*R;
   int    N    = (int)(0.5*rout/r) + 2;
   int    ium(0), ivm(0), iumAll(0), ivmAll(0), kount(0), ntot(0), nin(0);
-  std::vector<int>  ntype(3,0);
+  std::vector<int>  ntype(6,0);
```

Comment:
ntype is used only in debug mode. Please, move it to in between the preprocessor directives.